From 04bdc10311d634b5b57f7182fcc2d69473df0bd3 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Fri, 29 Dec 2023 09:56:25 +0100
Subject: [PATCH] ServiceAgent.h: Fix HandlerRequestTypes

It has a typo, when fixing the typo it complains about redefinition, so let's define it with a proper name.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 include/ServiceAgent.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/ServiceAgent.h b/include/ServiceAgent.h
index dafedd0..d770d59 100644
--- a/include/ServiceAgent.h
+++ b/include/ServiceAgent.h
@@ -118,7 +118,7 @@ enum RequestState {
 };
 
 // Mapped with HandlerTypes
-enum HanlderRequestType {
+enum HandlerType {
     HANDLER_HYBRID_BIT = (1u << HANDLER_HYBRID),
     HANDLER_GPS_BIT = (1u << HANDLER_GPS),
     HANDLER_NETWORK_BIT = (1u << HANDLER_NETWORK)
